src/assets/translations/en/main.json

```diff
@@ -1814,7 +1814,8 @@
     "importAccounts": {
       "title": "Import %{chainNamespaceDisplayName} Accounts",
       "description": "Select the accounts you would like to import from the list below.",
-      "import": "Import"
+      "import": "Import",
+      "loadMoreDisabled": "This wallet has no more accounts to import for this chain."
```

Comment:
```suggestion
      "loadMoreDisabled": "This wallet does not support multiple accounts for this chain."
```

Reply:
Precisely this ☝🏽 multi-account is virtually infinite, the reason why loading more is not supported isn't because there isn't more to load per se (assuming you have the same seed in MM and native, you could load an infinite number of accounts).
But rather than deriving multi-account isn't supported for these chains with MM (and MM impersonators)/ multichainsnap.

Reply:
Also we probably want to give this a more semantic name re: multi-account unsupported